pyemma/msm/estimators/maximum_likelihood_hmsm.py

```diff
@@ -204,54 +206,56 @@ def _estimate(self, dtrajs):
 
         # INIT HMM
-        from bhmm import init_discrete_hmm
+        from deeptime.markov.hmm import init
         from pyemma.msm.estimators import MaximumLikelihoodMSM
         from pyemma.msm.estimators import OOMReweightedMSM
-        if self.msm_init=='largest-strong':
-            hmm_init = init_discrete_hmm(dtrajs_lagged_strided, self.nstates, lag=1,
-                                         reversible=self.reversible, stationary=True, regularize=True,
-                                         method='lcs-spectral', separate=self.separate)
-        elif self.msm_init=='all':
-            hmm_init = init_discrete_hmm(dtrajs_lagged_strided, self.nstates, lag=1,
-                                         reversible=self.reversible, stationary=True, regularize=True,
-                                         method='spectral', separate=self.separate)
-        elif isinstance(self.msm_init, (MaximumLikelihoodMSM, OOMReweightedMSM)):  # initial MSM given.
-            from bhmm.init.discrete import init_discrete_hmm_spectral
-            p0, P0, pobs0 = init_discrete_hmm_spectral(self.msm_init.count_matrix_full, self.nstates,
-                                                       reversible=self.reversible, stationary=True,
-                                                       active_set=self.msm_init.active_set,
-                                                       P=self.msm_init.transition_matrix, separate=self.separate)
-            hmm_init = bhmm.discrete_hmm(p0, P0, pobs0)
-            observe_subset = self.msm_init.active_set  # override observe_subset.
-        else:
-            raise ValueError('Unknown MSM initialization option: ' + str(self.msm_init))
-
-        # ---------------------------------------------------------------------------------------
-        # Estimate discrete HMM
-        # ---------------------------------------------------------------------------------------
-
-        # run EM
-        from bhmm.estimators.maximum_likelihood import MaximumLikelihoodEstimator as _MaximumLikelihoodEstimator
-        hmm_est = _MaximumLikelihoodEstimator(dtrajs_lagged_strided, self.nstates, initial_model=hmm_init,
-                                              output='discrete', reversible=self.reversible, stationary=self.stationary,
-                                              accuracy=self.accuracy, maxit=self.maxit)
-        # run
-        hmm_est.fit()
-        # package in discrete HMM
-        self.hmm = bhmm.DiscreteHMM(hmm_est.hmm)
+        from threadpoolctl import threadpool_limits
+        with threadpool_limits(limits=1):
+            if self.msm_init == 'largest-strong':
+                hmm_init = init.discrete.metastable_from_data(dtrajs, n_hidden_states=self.nstates, lagtime=self.lag,
+                                                              stride=self.stride, mode='largest-regularized',
+                                                              reversible=self.reversible, stationary=True,
```

Comment:
`stationary` flag is set to False in the constructor - should be `self.stationary` or remove flag I suppose

Reply:
Or not because that's just the initial HMM?

Reply:
This is just reproducing the old behavior -- whether stationary is true or false should optimally be up to the user but the pyemma api currently doesn't allow that